### ISSUE TITLE
Should require-valid-alt-text rule require a role="presentation/none" when the alt text is blank like alt=""

### ISSUE THREAD

**Issue by lucasmerat** [ember-template-lint/ember-template-lint#2579]
I noticed the linter fails for [require-valid-alt-text](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/require-valid-alt-text.md) when adding alt="" without one of these roles. 

Based on W3C: "A null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers.

Screen readers also allow the use of WAI-ARIA to hide elements by using role="presentation". However, currently, this feature is not as widely supported as using a null alt attribute."

And you can see a number of decorative images with the alt="" attribute and no presentation/none role attribute.

Source: https://www.w3.org/WAI/tutorials/images/decorative/

So my question is, should it be changed to allow a simple alt="" without one of these two presentation/none roles added?

**Comment by nicolasgasco** [comment]
Hi, I have exactly the same doubt as @lucasmerat and would love to hear your 2 cents on this. Feels like that rule might be a bit too strict.
